src/main/webapp/import-editor.xed

```diff
@@ -765,22 +766,30 @@
 
   <xed:template id="projectInformation">
     <script src="{$WebApplicationBaseURL}js/project_autocomplete.js" />
+    <xed:bind xpath="mods:extension/temporary_search_input">
+      <div class="form-group form-inline">
+        <label for="{xed:generate-id()}" class="mycore-form-label">
+          <xed:output i18n="ubo.project.search" />:
+        </label>
+        <input id="project_search" type="text" class="mycore-form-input" />
+    </div>
+    </xed:bind>
     <xed:bind xpath="mods:extension[@displayLabel='project']">
       <xed:bind xpath="cerif:Project">
         <xed:bind xpath="cerif:Acronym">
           <div class="form-group form-inline">
             <label for="{xed:generate-id()}" class="mycore-form-label">
               <xed:output i18n="ubo.project.acronym" />:
             </label>
-            <input id="project_acronym" type="text" class="mycore-form-input" />
+            <input id="project_acronym" readonly="readonly" type="text" class="mycore-form-input" />
```

Comment:
we should use bootstrap css classes here, see https://getbootstrap.com/docs/4.4/components/forms/#readonly
but atm they doesn't work well with css class "mycore-form-input"

Reply:
"the readonly boolean attribute"
<input class="form-control" type="text" placeholder="Readonly input here..." readonly>
This is what im currently doing. This is NOT a CSS class.
If we want to make the users aware that this is a "readonly"-field, we can insert some text there "this is not for editing, use the search field instead". I will implement it like that.

Reply:
Ah I see, it behaves a little different to the Bootstrap 4 standart, when focusing the readonly field it gets "active" (white) as if you can edit it. I will change this instead.